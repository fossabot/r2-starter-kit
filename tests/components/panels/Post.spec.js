import React from 'react';
import { shallow } from 'enzyme';

import Component from 'components/panels/Post';

describe('<Post />', () => {
  const initProps = {
    uiState: 'idle',
  };

  let wrapper;
  beforeAll(() => {
    wrapper = shallow(<Component {...initProps} />);
  });

  it('should render initial page', () => {
    expect(wrapper).toMatchSnapshot();
  });

  it('should render component with uiState prop which value is "loading"', () => {
    wrapper.setProps({ uiState: 'loading' });
    expect(wrapper).toMatchSnapshot();
  });
});
