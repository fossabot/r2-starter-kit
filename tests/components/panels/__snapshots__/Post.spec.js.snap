// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Post /> should render component with uiState prop which value is "loading" 1`] = `
<Fragment>
  <Loading />
</Fragment>
`;

exports[`<Post /> should render initial page 1`] = `
<Fragment>
  <Item
    breadcrumb={Object {}}
    content={Object {}}
  />
</Fragment>
`;
